File: Python PowerPoint Examples/ShapesApiDemo/deleteSlideParagraphs.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "DeleteSlideParagraphs.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "4"
paragraphs = "[0,1]"
result =api.delete_slide_paragraphs(fileName, slide_index=slide_index, shape_index=shape_index, paragraphs=paragraphs,password=password, storage=storage,folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/deleteSlideParagraphs.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "DeleteSlideParagraphs.pptx";
$password = null;
$folder = "input";
$storage = null;

$slideIndex = 0;
$shapeIndex = "4";
$paragraphs = "0,1";
$apiInstance->deleteSlideParagraphs($name, $slideIndex, $shapeIndex, $paragraphs, $password, $folder, $storage);